src/utils/dmg.c

```diff
@@ -852,7 +852,7 @@ static int
 help_hdlr(int argc, char *argv[])
 {
 	printf("\
-usage: dmg COMMAND [OPTIONS]\n\
+usage: dmg_old COMMAND [OPTIONS]\n\
```

Comment:
It would be good to update this to note that the tool is deprecated, at a minimum. Even better would be printing a deprecation warning on stderr, but that might cause problems for scripts.

Reply:
I don't really want to change behaviour at this point, I think renaming to _old should suffice in addition to communication over the mailing lists